Makefile

```diff
@@ -1,20 +1,72 @@
 SHELL := /bin/bash
 .SHELLFLAGS = -euo pipefail -c
-MAKEFLAGS = -s
-CGO_ENABLED = 1
+CGO_ENABLED = 0
 BUILDTAGS :=
-BUILDFLAGS := -ldflags '-linkmode=external'
+BUILD_PATH:=build
+BUILD:=ascli
+ifeq ($(CGO_ENABLED),1)
+BUILDFLAGS += -linkmode=external
+else
+BUILDFLAGS :=
+endif
+GOOS           :=$(shell go env GOOS)
+GOARCH         :=$(shell go env GOARCH)
+GOMODULECMD    :=main
+
+
+SEMVER_VERSION    ?=3.0.0
```

Comment:
I wonder if it would make sense to parse this out from https://github.com/cloudfoundry/app-autoscaler-cli-plugin/blob/308b1c1138a19459c512a00868888e96f62fbe65/main.go#L19-L21